webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala

```diff
@@ -313,7 +314,7 @@ object DisplayDirectiveTree extends Loggable {
           }
         }
 
-        val directiveTagsListHtml = <div>{directive.tags.tags.map(tag => <span class="tags-label"><i class="fa fa-tag"></i> <span class="tag-key">{tag.name.value}</span><span class="tag-separator"> = </span><span class="tag-value">{tag.value.value}</span></span>)}</div>
+        val directiveTagsListHtml = <div>{directive.tags.tags.map(tag => <span class="tags-label"><i class="fa fa-tag"></i> <span class="tag-key">{escapeHTML(tag.name.value)}</span><span class="tag-separator"> = </span><span class="tag-value">{escapeHTML(tag.value.value)}</span></span>)}</div>
```

Comment:
are you sure it needs to be fixed?

Reply:
I'm not sure where it's used, but it's clrearly not espapted, directive is the usual scala object